mssql_test.go

```diff
@@ -1799,6 +1799,46 @@ func TestMSSQLNextResultSet(t *testing.T) {
 		})
 }
 
+func TestMSSQLNextResultSetWithDifferentColumnsInResultSets(t *testing.T) {
```

Comment:
I thought I tried this myself, but, perhaps, I am mistaken. I will try again.

Reply:
I got an error in your original implementation something with unexpected number of columns in a scan or something to that extent.